localisation/00_zuwc_united_localisation.csv

```diff
@@ -2885,10 +2885,10 @@ EVTOPT_MNM_4018;Cognition is what I live for.;;Erkenntnis ist das, wof
 String_Death_is_coming;Death is coming;;Der Tod kommt;;;;;;;;;;;x
 GetDinnerAndDrinks_dinner_and_drinks;dinner and plenty of drinks;;Abendessen und viele Getr�nke;;;;;;;;;;;x
 GetDinnerAndDrinks_meeting;a meeting;;ein Meeting;;;;;;;;;;;x
-indulgence_ask;The High Priest will hear our plea for salvation.;Le Grand Pr�tre entendra notre plaidoyer pour le salut.;Der Hohepriester wird unsere Bitte um Erl�sung erh�ren.;;El Sumo Sacerdote oir� nuestra petici�n de salvaci�n.;;;;;;;;;x
-repentance_ask;The High Priest will hear our plea for salvation.;Le Grand Pr�tre �coutera notre demande de pardon.;Der Hohepriester wird unser Flehen um Erl�sung anh�ren.;;El Sumo Sacerdote escuchar� nuestra oraci�n de salvaci�n.;;;;;;;;;x
-EVTDESC39253;The High Priest has agreed to lift the excommunication against me!;Le Grand Pr�tre a accept� de lever mon excommunication!;Der Hohepriester hat zugestimmt, meine Exkommunikation aufzuheben!;;�El Sumo Sacerdote ha accedido a revocar la excomuni�n contra mi!;;;;;;;;;x
-EVTDESC39276;The High Priest wishes to inform us that we will now also obey an early liege.;Le Grand Pr�tre souhaite nous annoncer que nous ob�irons aussi � un jeune Seigneur.;Der Hohepriester w�nscht, uns dar�ber in Kenntnis zu setzen, dass wir uns nun auch einem fr�heren Lehnsherren unterwerfen werden m�ssen.;;El Sumo Sacerdote desea informarnos de que ahora tambi�n deberemos obedecer a un se�or terrenal.;;;;;;;;;x
+indulgence_ask;The [Root.RelHead.GetTitle] will hear our plea for salvation.;Le [Root.RelHead.GetTitle] entendra notre plaidoyer pour le salut.;Der [Root.RelHead.GetTitle] wird unsere Bitte um Erl�sung erh�ren.;;El [Root.RelHead.GetTitle] oir� nuestra petici�n de salvaci�n.;;;;;;;;;x
+repentance_ask;The [Root.RelHead.GetTitle] will hear our plea for salvation.;Le [Root.RelHead.GetTitle] �coutera notre demande de pardon.;Der [Root.RelHead.GetTitle] wird unser Flehen um Erl�sung anh�ren.;;El [Root.RelHead.GetTitle] escuchar� nuestra oraci�n de salvaci�n.;;;;;;;;;x
+EVTDESC39253;The [Root.RelHead.GetTitle] has agreed to lift the excommunication against me!;Le [Root.RelHead.GetTitle] a accept� de lever mon excommunication!;Der [Root.RelHead.GetTitle] hat zugestimmt, meine Exkommunikation aufzuheben!;;�El [Root.RelHead.GetTitle] ha accedido a revocar la excomuni�n contra mi!;;;;;;;;;x
```

Comment:
Looks like this localization isn't used anymore in the vanilla. These event not called anywhere. I would delete it.

Reply:
Meh, no, fine, keep it. Who knows? Maybe it's used somewhere.

Reply:
Alright, I'll keep it.